README.md

```diff
@@ -28,7 +28,7 @@ languageXchange is an application built with Angular and Ionic that helps users
 
 ## Status
 
-This project is currently in Beta phase v0.X.x For more information, please refer to the project's GitHub repository.
+This project is currently in Beta phase v0.5.17 For more information, please refer to the project's GitHub repository.
```

Comment:
Can you please change with `v0.5.x`

why: _because no need to change every minor update_